acceptance/bundle/validate/interpolation-in-auth-field/databricks.yml

```diff
@@ -0,0 +1,10 @@
+bundle:
```

Comment:
We have bundle/variables for this type of tests and a similar tests exists - https://github.com/databricks/cli/pull/2117

Reply:
Fair enough, removed the new test. I added that in to test other fields that `host` but it's not really necessary.